elm/wizard.py

```diff
@@ -152,10 +160,10 @@ def chat(self, query,
              token_budget=None,
              new_info_threshold=0.7,
              print_references=False,
-             return_chat_obj=False):
+             return_chat_obj=False,
+             timeit=False):
```

Comment:
why don't we just always have timeit=True? I think we should just remove this as a kwarg and always return performance metrics. This will break anything that expects a certain number of outputs but the compute is basically free and it's useful information.